### DISCUSSION TITLE
Roy ve Profil Fotoğrafı

### DISCUSSION THREAD

**Discussion by KayraMertAydin** [sonsuzus/sonsuzus#430] [Soru Cevap]
Roy, Facebook'taki profil resmini değiştirmek istiyor. Artık Facebook'un yükleyebileceğimiz resmin boyutu konusunda bazı kısıtlamaları var.
Resmin minimum boyutu L x L olabilir; burada L, karenin kenar uzunluğudur.

Artık Roy'un çeşitli boyutlarda N fotoğrafı var.
Bir fotoğrafın boyutu G x Y olarak gösterilir
burada W - fotoğrafın genişliği ve H - fotoğrafın yüksekliği

Herhangi bir fotoğraf yüklendiğinde aşağıdaki olaylar meydana gelebilir:

[1] Genişlik veya yükseklikten herhangi biri L'den küçükse, kullanıcıdan bir tane daha yüklemesi istenir. Bu durumda "BAŞKA BİR YÜKLE" yazdırın.
[2] Genişlik ve yükseklik her ikisi de yeterince büyükse ve
(a) Fotoğraf zaten kare ise kabul edilir. Bu durumda "KABUL EDİLDİ" yazdırın.
(b) aksi takdirde kullanıcıdan onu kırpması istenir. Bu durumda "CROP IT" yazdırın.

(alıntılar sadece açıklama amaçlıdır)

Giriş olarak L, N, W ve H verildiğinde, uygun metni çıktı olarak yazdırın.

Giriş:
İlk satırda L var.
İkinci satır N, fotoğraf sayısını içerir.
Aşağıdaki N satırların her biri boşlukla ayrılmış iki tam sayı W ve H içerir.

Çıktı:
Her fotoğraf için uygun metni yeni bir satıra yazdırın.

Kısıtlamalar:
1 <= U,G,Y <= 10000
1 <= N <= 1000

**Comment by KayraMertAydin** [comment]
https://www.hackerearth.com/practice/basic-programming/input-output/basics-of-input-output/practice-problems/algorithm/roy-and-profile-picture/

**Comment by Erva-Naz** [comment]
#include <iostream>
using namespace std;
int main() {
	int l, n,w,h;

	cin>>l;
	cin>>n;
	for(int i=0;i<n;i++) {
		cin>>w>>h;

		if(w<l||h<l) {
			cout << "UPLOAD ANOTHER"<<endl; 
		}

		else if(w==h) {
			cout <<"ACCEPTED"<<endl;
		}

		else{
			cout << "CROP IT"<<endl;
		}
	}

	

}
